spec/assigner_spec.rb

```diff
@@ -40,6 +40,23 @@
     a.issue_ids.count.should == 1
   end
 
+  it 'should assign comment with ticket when toggled from redmine' do
+    a = Assigner.new '92345678: Feature #7682 Implement toggl directly from redmine'
+    a.issue_ids.first.should == 7682
+    a.issue_ids.count.should == 1
```

Comment:
Haha, yes and no - it is just the old syntax and i do not want to rewrite the whole code. Absolutely agree with you considering the context parts, but same reason there.

If you feel like you can rewrite / restructure the tests in your 20% time :wink: